tests/db_access/prepared_statements.cpp

```diff
@@ -567,6 +568,79 @@ go_bandit([](){
+
+            AssertThat(id, Equals(42));
+            AssertThat(blob.size(), Equals(5u));
+            AssertThat(binary.size(), Equals(10u));
+            AssertThat(varbinary.size(), Equals(5u));
+            AssertThat(blob.getStringView()=="ab0cd"s, IsTrue());
+            AssertThat(binary.getStringView(), Equals("ef\0gh\0\0\0\0\0"s));
+            AssertThat(varbinary.getStringView(), Equals("ij\0kl"s));
+        });
+
+        it("can work with psReadValues (passing PreparedStatement object)", [&](){
+            int id{};
+            BlobData blob{}, binary{}, varbinary{};
+            auto preparedStatement = connection.makePreparedStatement<ResultBindings<int, BlobData, BlobData, BlobData>>(
+                "SELECT `id`, `blob`, `binary`, `varbinary` FROM `test_superior_sqlpp`.`binary_data` ORDER BY `id` LIMIT 1"
+            );
+            psReadValues(preparedStatement, id, blob, binary, varbinary);
+
+            AssertThat(id, Equals(42));
+            AssertThat(blob.size(), Equals(5u));
+            AssertThat(binary.size(), Equals(10u));
+            AssertThat(varbinary.size(), Equals(5u));
+            AssertThat(blob.getStringView()=="ab0cd"s, IsTrue());
+            AssertThat(binary.getStringView(), Equals("ef\0gh\0\0\0\0\0"s));
+            AssertThat(varbinary.getStringView(), Equals("ij\0kl"s));
+        });
+
+        it("can work with psReadValues (throws multiple rows error)", [&](){
+            int id{};
+            BlobData blob{}, binary{}, varbinary{};
+            AssertThrows(UnexpectedMultipleRowsError,
+                psReadValues("SELECT `id`, `blob`, `binary`, `varbinary` FROM `test_superior_sqlpp`.`binary_data` ORDER BY `id`", connection, id, blob, binary, varbinary)
+            );
+        });
+
+        it("can work with psReadQuery (valid types, passing PreparedStatement object)", [&](){
+            auto preparedStatement = connection.makePreparedStatement<ResultBindings<int, BlobData, BlobData, BlobData>>(
+                "SELECT `id`, `blob`, `binary`, `varbinary` FROM `test_superior_sqlpp`.`binary_data` ORDER BY `id` LIMIT 1"
+            );
+
+            psReadQuery(preparedStatement, [&](int id, const BlobData &blob, const BlobData &binary, const BlobData &varbinary) {
+                AssertThat(id, Equals(42));
+                AssertThat(blob.size(), Equals(5u));
+                AssertThat(binary.size(), Equals(10u));
+                AssertThat(varbinary.size(), Equals(5u));
+                AssertThat(blob.getStringView()=="ab0cd"s, IsTrue());
+                AssertThat(binary.getStringView(), Equals("ef\0gh\0\0\0\0\0"s));
+                AssertThat(varbinary.getStringView(), Equals("ij\0kl"s));
+            });
+        });
+
+        it("can work with psReadQuery (valid types, passing query string)", [&](){
```

Comment:
This kind of test is already here, few lines above. But what is missing is query with multiple lines result.